packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -78,12 +79,26 @@ export class StatusBarImpl extends ReactWidget implements StatusBar {
     constructor(
         @inject(CommandService) protected readonly commands: CommandService,
         @inject(LabelParser) protected readonly entryService: LabelParser,
-        @inject(FrontendApplicationStateService) protected readonly applicationStateService: FrontendApplicationStateService
+        @inject(FrontendApplicationStateService) protected readonly applicationStateService: FrontendApplicationStateService,
+        @inject(PreferenceService) protected readonly preferences: PreferenceService,
```

Comment:
@EstherPerelman please add an entry to the changelog for the breaking change. Adding new dependencies via the constructor is a form of breaking change according to our guidelines:
- https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#property-injection